src/app/patient-dashboard/formentry/formentry.component.spec.ts

```diff
@@ -504,7 +508,7 @@ describe('Component: FormentryComponent', () => {
                 expect(formFactory.createForm).toHaveBeenCalled();
                 // now check to ensure we are setting default data
                 expect(userDefaultPropertiesService.getCurrentUserDefaultLocationObject)
-                    .not.toHaveBeenCalled();
+                  .toHaveBeenCalled();
```

Comment:
modify this test to actually ensure that the form is not populated with default values in edit mode. It is an important test. Talk to Allan about it.